packages/core/src/modules/credentials/protocol/v2/messages/V2RequestCredentialMessage.ts

```diff
@@ -0,0 +1,52 @@
+import { Expose, Type } from 'class-transformer'
```

Comment:
I assume you mean RevocationNoficationMessage, rather than RequestCredentialMessage.

Moved the two messages and two handlers into folders unter protocol/revocation-notification